src/features/campaigns/components/CampaignActionButtons.tsx

```diff
@@ -79,6 +84,24 @@ const CampaignActionButtons: React.FunctionComponent<
       },
     });
   };
+
+  const handleCreateEvent = () => {
+    const defaultStart = new Date();
+    defaultStart.setDate(defaultStart.getDate() + 1);
+
+    const defaultEnd = new Date(defaultStart.getTime() + 60 * 60 * 1000);
+
+    eventModel.createEvent(
+      //TO DO:give null to ids when API supports it.
```

Comment:
Great that you make a comment like this. But I think it needs to be `TODO` (one word) for tools to find it, at least that's what I manually search for when looking for todo comments.